packages/trader/src/Modules/Account/Sections/Verification/ProofOfAddress/proof-of-address-container.jsx

```diff
@@ -35,7 +35,7 @@ class ProofOfAddressContainer extends React.Component {
                     status: document.status,
                     needs_poi,
                     is_loading: false,
-                    submitted_poa: !(needs_verification.length && needs_verification.includes('document')),
+                    submitted_poa: needs_verification.length && needs_verification.includes('document'),
```

Comment:
if there is document in needs_verification means that the client required to submit poa, not submitted poa afaik

Reply:
okay, so is it better to just check for `document.status === 'pending'`